policyengine_us/variables/gov/states/dc/dhs/tanf/income/dc_tanf_countable_income_for_computing_benefits.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class dc_tanf_countable_income_for_computing_benefits(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "DC TANF countable earned income"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.DC
+
+    def formula(spm_unit, period, parameters):
+        gross_earnings = spm_unit("dc_tanf_gross_earned_income", period)
+        gross_unearnings = spm_unit(
```

Comment:
Unearnings deserves to be a word but it sadly is not
```suggestion
        gross_unearned_income = spm_unit(
```